examples/tv-casting-app/APIs.md

```diff
@@ -810,21 +880,280 @@ let kDesiredEndpointVendorId: UInt16 = 65521;
+
+ConnectionCallbacks connectionCallbacks =
+    new ConnectionCallbacks(
+        new MatterCallback<Void>() {
+            @Override
+            public void handle(Void v) {
+                Log.i(
+                    TAG,
+                    "Successfully connected to CastingPlayer with deviceId: "
+                        + targetCastingPlayer.getDeviceId());
+                getActivity()
+                    .runOnUiThread(
+                        () -> {
+                        connectionFragmentStatusTextView.setText(
+                            "Successfully connected to Casting Player with device name: "
+                                + targetCastingPlayer.getDeviceName()
+                                + "\n\n");
+                        connectionFragmentNextButton.setEnabled(true);
+                        });
+            }
+        },
+        new MatterCallback<MatterError>() {
+            @Override
+            public void handle(MatterError err) {
+                Log.e(TAG, "CastingPlayer connection failed: " + err);
+                getActivity()
+                    .runOnUiThread(
+                        () -> {
+                        connectionFragmentStatusTextView.setText(
+                            "Casting Player connection failed due to: " + err + "\n\n");
+                        });
+            }
+        },
+        new MatterCallback<CommissionerDeclaration>() {
+            @Override
+            public void handle(CommissionerDeclaration cd) {
+            getActivity()
+                .runOnUiThread(
+                    () -> {
+                        connectionFragmentStatusTextView.setText(
+                            "CommissionerDeclaration message received from Casting Player: \n\n");
+                        if (cd.getCommissionerPasscode()) {
+
+                        displayPasscodeInputDialog(getActivity());
+                        ...
+
+                        // Update the commissioning session's PAKE verifier with the user-entered Passcode
+                        InitializationExample.commissionableDataProvider.updateCommissionableDataSetupPasscode(
+                                        passcodeLongValue, DEFAULT_DISCRIMINATOR_FOR_CGP_FLOW);
+
+                        // Call continueConnecting to complete commissioning.
+                        MatterError err = targetCastingPlayer.continueConnecting();
+                        ...
+                        }
+                    });
+            }
         }
-    }, desiredEndpointFilter: desiredEndpointFilter)
+    );
+
+MatterError err =
+    targetCastingPlayer.verifyOrEstablishConnection(
+        connectionCallbacks, MIN_CONNECTION_TIMEOUT_SEC, idOptions);
+...
+```
+
+On iOS, the Casting Client can connect to a `CastingPlayer`, using the optional
+`CastingPlayer` / Commissioner-Generated Passcode UDC feature, by successfully
+calling `verifyOrEstablishConnection`, updating the PAKE verifier and then
```

Comment:
Same comment as above, just say "updating the set up passcode" because that's what the client is visibly doing.